kipoiseq/extractors/vcf.py

```diff
@@ -90,6 +93,22 @@ def query_variants(self, intervals, sample_id=None, progress=False):
                  for i in intervals]
         return VariantIntervalQueryable(self, pairs, progress=progress)
 
+    def query_all(self, progress=False):
+        """Convert all variants into queryable object without interval so
+        interval filters will not work.
+
+        # Example
+            To fetch variants if only single variant present in interval.
```

Comment:
"To fetch variants if there is only a single variant present in interval:"
Better?